sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestBatching.java

```diff
@@ -67,6 +67,21 @@ public void testInsertSingleIteratorNoTx() {
         assertThat(b.size()).isEqualTo(2);
     }
 
+    @Test
+    public void testBindConstantNull() {
+        UsesBatching b = handle.attach(UsesBatching.class);
```

Comment:
avoid single-letter variables please (aside from the meme `i` for iterations, I'd say `Handle h` at most could also be a meme in our context)

Reply:
Ah, this is all just clone of existing tests, with only one other change...

Reply:
Sounds like the improvement could be made in 2 places :)